packages/vm/src/instance.rs

```diff
@@ -562,7 +562,7 @@ mod test {
 
         let report2 = instance.create_gas_report();
         assert_eq!(report2.used_externally, 134);
-        assert_eq!(report2.used_internally, 66495);
+        assert_eq!(report2.used_internally, 66539);
```

Comment:
All those changes to the error types and the gas barely changes? I was expected a bigger diff (plus or minus). I guess the error path was not executed here however.

Curious about the change to binary size.

Reply:
Yeah, error paths are probably not executed much in the gas tests. Binary got a 7% smaller in this PR:
`-12.2 KB (93%) packages/vm/testdata/contract_0.11.wasm`